docker/Dockerfile

```diff
@@ -13,5 +13,5 @@ RUN ["npm", "install"]
 RUN ["npx", "witnet-toolkit", "update"]
 
 # Set the entrypoint
-ENTRYPOINT ["python", "price_feed_poller.py", "--loop_interval_secs", "90", "--config_file"]
+ENTRYPOINT ["python", "price_feeds_poller.py", "--loop_interval_secs", "45", "--toml_file"]
 CMD ["$1"]
```

Comment:
```suggestion
ENTRYPOINT ["python", "price_feeds_poller.py", "--loop_interval_secs", "45"]
CMD ["--toml_file", "$1"]
```

Reply:
Done in https://github.com/witnet/price_feed_poller/pull/32/commits/d89fb3cba95c049e50f9a625366d8d2aa92609e1.